src/sql/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -119,6 +122,14 @@ function isWelcomePageEnabled(configurationService: IConfigurationService, conte
 	return startupEditor.value === 'welcomePage' || startupEditor.value === 'readme' || startupEditor.value === 'welcomePageInEmptyWorkbench' && contextService.getWorkbenchState() === WorkbenchState.EMPTY;
 }
 
+function isGuidedTourEnabled(configurationService: IConfigurationService) {
```

Comment:
Make this return a boolean if the function has the word 'is'

Reply:
I believe this has been fixed